packages/edit-site/src/components/layout/index.js

```diff
@@ -277,6 +277,20 @@ export default function Layout() {
 							{ canvasResizer }
 							{ !! canvasSize.width && (
 								<motion.div
+									whileHover={
+										canvasMode === 'view'
```

Comment:
I think this condition is not complete as I see the hover effect even in the "browse all templates" page. We should probably use the same condition as the one enabling the click in canvas. Probably needs an `isEditorPage && canvasMode === 'view'`